src/js/containers/ClientLoginPage/ClientLoginPage.js

```diff
@@ -44,42 +44,44 @@ class ClientLoginPage extends Component {
-                            </LayoutColumn>
-                        </LayoutRow>
-                    </LayoutContainer>
-                </Form>
+            <div>
+                <div id="cf-login-page">
+                    <Form layout="vertical" onSubmit={(e) => this.handleLoginSubmit(e)}>
+                        <LayoutContainer>
+                            <LayoutRow>
+                                <LayoutColumn width={1/1}>
+                                    <FormHeader title={formatMessage({ id: 'component.clientLogin.form.title' })} style={{ textAlign:'center' }}/>
+                                </LayoutColumn>
+                            </LayoutRow>
+                                <FormFieldset legend="">
+                                    <LayoutRow>
+                                        <LayoutColumn width={1/1}>
+                                            <FormLabel hidden><FormattedMessage id="component.clientLogin.form.email"/></FormLabel>
+                                            <Input name="email" type="text" value={this.state.email} onChange={this.handleEmailChange.bind(this)} placeholder={formatMessage({ id: 'component.clientLogin.form.email' })}/>
+                                        </LayoutColumn>
+                                    </LayoutRow>
+                                    <LayoutRow>
+                                        <LayoutColumn width={1/1}>
+                                            <FormLabel hidden><FormattedMessage id="component.clientLogin.form.apiKey"/></FormLabel>
+                                            <Input name="apiKey" type="text" value={this.state.apiKey} onChange={this.handleApiKeyChange.bind(this)} placeholder={formatMessage({ id: 'component.clientLogin.form.apiKey' })}/>
+                                        </LayoutColumn>
+                                    </LayoutRow>
+                                    <LayoutRow>
+                                        <LayoutColumn width={1/1}>
+                                            <Button submit type="success" onClick={(e) => this.handleLoginSubmit(e)}>
+                                                <FormattedMessage id="component.clientLogin.form.button" />
+                                            </Button>
+                                        </LayoutColumn>
+                                    </LayoutRow>
+                                </FormFieldset>
+                            <LayoutRow>
+                                <LayoutColumn width={1/1}>
+                                    <p style={{ textAlign: 'center', marginBottom: '2.5rem' }}><FormattedMessage id="component.clientLogin.cloudflare.description"/> <a href={signupLinkWithUTM} target="_blank">CloudFlare.com</a>.</p>
+                                </LayoutColumn>
+                            </LayoutRow>
+                        </LayoutContainer>
+                    </Form>
+                </div>
```

Comment:
The "fix" is really in the MarketingFeatureCollection.  I moed it outside the `.cf-login-page` because if we style based on that class in hacks.css I don't want those styles applying to the marketing features.